io.github.Andrettin.Wyrmsun.yml

```diff
@@ -0,0 +1,92 @@
+app-id: io.github.Andrettin.Wyrmsun
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: wyrmsun
+rename-icon: wyrmsun
+rename-desktop-file: wyrmsun.desktop
+rename-appdata-file: wyrmsun.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --persist=.stratagus
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
```

Comment:
Would it work with LuaJIT?

Reply:
There is no indication in cmake or https://github.com/Andrettin/Wyrmgus/blob/master/doc/install.html